presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTask.java

```diff
@@ -453,18 +459,20 @@ public URI getRemoteTaskLocation()
     @Override
     public void start()
     {
-        try (SetThreadName ignored = new SetThreadName("HttpRemoteTask-%s", taskId)) {
-            // to start we just need to trigger an update
-            started.set(true);
-            scheduleUpdate();
-
-            taskStatusFetcher.start();
-            taskInfoFetcher.start();
-        }
+        taskEventLoop.execute(() -> {
```

Comment:
Does this change remove all synchronized sections?

Reply:
@arhimondr yes, all synchronized sections are now async. The only thing I am not certain is this change https://github.com/prestodb/presto/pull/24288/files#r1894673304. The original synchronized implementation might return a future but now we will need to notify the listener in a async way. I wonder what your thought is on this. So far, I am testing the code on verifier and things look good.